monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/PluginTable.tsx

```diff
@@ -54,17 +104,26 @@ const renderSafetyCell = (params) => {
   );
 }
 
+export type PluginRow = {
+  id: string,
+  name: string,
+  version: string,
+  pluginType: string,
+  author: string,
+  description: string,
+  safe: boolean
+};
 
-export const getPluginsTableRows = (pluginsList :AgentPlugin[]) => {
+export const getDefaultPluginsTableRows = (pluginsList: AgentPlugin[]) :PluginRow[] => {
```

Comment:
What is the meaning of default here? Based on the name it sounds like it would output a default set of rows, instead of generating rows from the data with the default set of columns. Maybe `'defaultGetPluginsTableRows` makes more sense?